### PR TITLE
Merge latest from Sufia

### PR DESCRIPTION
**Pull request by mjgiarlo**
@projecthydra-labs/hyrax-code-reviewers

### CODE REVIEW

**Comment by mjgiarlo** [comment]
@jcoyne last travis build running. This is ready for review (in case I messed up the merge).

File: spec/services/hyrax/actor_factory_spec.rb [hunk 1]
```diff
@@ -13,9 +13,10 @@
                          Hyrax::Actors::AttachFilesActor,
                          Hyrax::Actors::ApplyOrderActor,
                          Hyrax::Actors::InterpretVisibilityActor,
+                         Hyrax::DefaultAdminSetActor,
+                         Hyrax::Actors::InitializeWorkflowActor,
                          Hyrax::ApplyPermissionTemplateActor,
-                         Hyrax::Actors::GenericWorkActor,
-                         Hyrax::Actors::InitializeWorkflowActor]
+                         Hyrax::Actors::GenericWorkActor]
```
Comment (jcoyne) [inline spec/services/hyrax/actor_factory_spec.rb]: This seems like a problem. Why did you move Initialize up 3 steps?

File: spec/services/hyrax/actor_factory_spec.rb [hunk 2]
```diff
@@ -30,9 +31,10 @@
         Hyrax::Actors::AttachFilesActor,
         Hyrax::Actors::ApplyOrderActor,
         Hyrax::Actors::InterpretVisibilityActor,
+        Hyrax::DefaultAdminSetActor,
+        Hyrax::Actors::InitializeWorkflowActor,
```
Comment (jcoyne) [inline spec/services/hyrax/actor_factory_spec.rb]: Why is Initialize moving up?

File: app/services/hyrax/actor_factory.rb [hunk 3]
```diff
@@ -9,9 +9,10 @@ def self.stack_actors(curation_concern)
        Hyrax::Actors::AttachFilesActor,
        Hyrax::Actors::ApplyOrderActor,
        Hyrax::Actors::InterpretVisibilityActor,
+       DefaultAdminSetActor,
+       Hyrax::Actors::InitializeWorkflowActor,
```
Comment (jcoyne) [inline app/services/hyrax/actor_factory.rb]: Why is Initialize moving up?  Don't we need to save the model before we activate the workflow?

Comment (mjgiarlo) [inline app/services/hyrax/actor_factory.rb]: The order of actors in Hyrax is different from the order of actors in Sufia. See https://github.com/projecthydra/sufia/blob/master/app/services/sufia/actor_factory.rb. I went with the order from Sufia where I've seen workflow working before rather than Hyrax. Do you think Sufia is currently wrong too?

**Comment by mjgiarlo** [comment]
@jcoyne Fixed. Re-review?
